code/line_commands.py

```diff
@@ -44,3 +44,7 @@ def camel_right():
     def line_clone(line: int):
         """Clones specified line at current position"""
 
+    def slap():
+        """Move cursor to the right by camel case/subword"""
+        actions.edit.line_end()
+        actions.key("enter")
```

Comment:
Quick flag that it was hard to decide if this belonged in `line_commands.py` or `edit.py` since they both had very similar (overlapping?) commands

Reply:
It should probably be in edit.py, since it has nothing to do with line numbers.